contexts/arbo-context.tsx

```diff
@@ -84,6 +93,28 @@ export const arboReducer = (state: ArboStateType, action: ArboAction) => {
   }
 };
 
+const adjustMapPositionIfCountryFilterHasChanged = (
+  action: ArboAction
+): void => {
+  if (action.payload.filter === "country") {
+    const allSelectedCountryBoundingBoxes = action.payload.value
+      .map((countryName: string) => getBoundingBoxFromCountryName(countryName))
+      .filter((boundingBox: CountryBoundingBox) => !!boundingBox);
+
+    if(allSelectedCountryBoundingBoxes.length === 0) {
+      action.payload.map.fitBounds([-180, -90, 180, 90]);
```

Comment:
Resets the position of the map to the equator + prime meridian if you deselect the country you have selected or if you've only selected countries that don't exist in `countryBoundingBoxData` (currently every country in pathotracker has been added to `countryBoundingBoxData` so there shouldn't be any problems but pointing it out just in case.